packages/ng-schematics/src/utils/cli-config.ts

```diff
@@ -41,17 +41,10 @@ function GetCliConfig(tree: Tree): Config {
 function getPort(workspace: WorkspaceSchema) {
 	const targetProjectName = workspace.defaultProject;
 	const projectServe = targetProjectName
-		? workspace.projects[targetProjectName].architect.serve.options
+		? (workspace.projects[targetProjectName].architect || { serve: { }}).serve.options
```

Comment:
Erm.. wut

Reply:
Typescript was going crazy on my end, I've reverted this